plugin/src/main/groovy/com/genymotion/GenymotionPluginExtension.groovy

```diff
@@ -241,17 +241,12 @@ class GenymotionPluginExtension {
     /**
      * Configuration management
      */
-
     def processConfiguration(GMTool gmtool = GMTool.newInstance()) {
-        project.genymotion.config.version = gmtool.getVersion()
-
         GenymotionConfig config = project.genymotion.config
         config.applyConfigFromFile(project)
-
         if (!config.isEmpty()) {
-
             gmtool.setConfig(config)
-
+            project.genymotion.config.version = gmtool.getVersion()
```

Comment:
This should be outside the if (line 250) as it is needed whatever the config is set or not.